admin/google_search_console/class-gsc-service.php

```diff
@@ -57,9 +57,14 @@ public function get_sites() {
 		$response_json = $this->client->do_request( 'sites', true );
 
 		// Do list sites request.
+		// Ignore coding standarsd for object properties.
+		// @codingStandardsIgnoreStart
```

Comment:
Those properties should pass as long as they are on the whitelist as added in https://github.com/Yoast/wordpress-seo/pull/6875/commits/14f775fc7a7342dbf14e42a93870bf457b679d36